sway/tree/container.c

```diff
@@ -1358,7 +1358,7 @@ void container_detach(struct sway_container *child) {
 		// We may have removed the last tiling child from the workspace. If the
 		// workspace layout was e.g. tabbed, then at this point it may be just
 		// H[]. So, reset it to the default (e.g. T[]) for next time.
-		if (!old_workspace->tiling->length) {
+		if (old_workspace->output && !old_workspace->tiling->length) {
```

Comment:
Can you please add to the comment above something like "If we are evacuating a workspace with only sticky floating containers, the workspace will already be detached from the output."?

Reply:
Done.